QuasistaticBrownianThermalNoise/QuasistaticBrownianThermalNoise.cpp

```diff
@@ -454,28 +464,82 @@ double ElasticProblem<dim>::getYijkl(int whichYijkl, unsigned int i,
   return 0.0;
 }
 
+// Switch case to find Yijkl using more than one loss angle for the coating.
+template <int dim>
+double ElasticProblem<dim>::getMultLossAngleYijkl(int whichYijkl,
+                                                  unsigned int i,
+                                                  unsigned int j,
+                                                  unsigned int k,
+                                                  unsigned int l) {
+  switch (whichYijkl) {
+    case (kAlGaAs):
+      return Y_AlGaAs_Phi[i][j][k][l];
+    case (kIso_AlGaAs):
+      return Y_Iso_AlGaAs_Phi[i][j][k][l];
+    case (kIso_Ta2O5):
+      return Y_Iso_Ta2O5_Phi[i][j][k][l];
+    case (kIso_FusedSilica):
+      return Y_Iso_FusedSilica_Phi[i][j][k][l];
+    default:
+      std::cout << "WARNING! Invalid Yijkl string. Do not trust results!\n";
+      break;
+  }
+  return 0.0;
+}
+
+// Calculates and returns the correct value for Y_ijkl in terms of the bulk
+// and shear modulus. The ints i,j,k,l that are passed are the ints from the for
+// loops that assign the correct values to the youngs tensor for each of the
+// coatings.
+double ScaleYikjlPhiBulk(double lame_mu, double lame_lambda, double phibulk,
```

Comment:
Do you think this function should be renamed `ScaleYijklPhiMu`, because it's really scaling terms proportional to lame_mu by the right thing?

Reply:
I think if we changed the names of the function, they should change from 

`ScaleYikjlPhiBulk  -> ScaleYikjlPhiLambda`
`ScaleYikjlPhiShear -> ScaleYikjlPhiMu`

since `ScaleYikjlPhiShear` scales by lambda and mu, while `ScaleYikjlPhiShear` only scales by mu.